.devcontainer/Dockerfile

```diff
@@ -6,3 +6,4 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
 # [Optional] Uncomment this section to install additional packages.
 # RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
 #     && apt-get -y install --no-install-recommends <your-package-list-here>
+
```

Comment:
is that newline on purpose?

Reply:
I noticed you set this to unmanaged. Should the file be deleted?

Reply:
Ah no, that was so I could add the apt-get update